README.md

```diff
@@ -229,3 +230,10 @@ Many thanks to all the [Scribe-Data contributors](https://github.com/scribe-org/
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
   <br>
 </div>
+
+<a id="architecture-diagrams"></a>
+# Architecture Diagrams  [`⇧`](#contents)
+Scribe[[1]](https://en.wikipedia.org/wiki/Scribe) is a community of language enthusiasts, learners and developers creating tools to help people communicate with confidence.We strive to be an inclusive community, welcoming coders and open-source supporters of any level.  
+ Here is the overview of the [architecture diagrams](https://github.com/scribe-org/Organization/blob/main/ARCHITECTURE.md#full-architecture) of our organization, the services, the applications, the processes and the external systems and sources with which it interacts. 
```

Comment:
When linking to the architecture diagrams, I'm thinking that it makes sense to simply link to the full file anyways.
So the `#full-architecture` in the url can get omitted here.